### ISSUE TITLE
Text copy-to-clipboard support

### ISSUE THREAD

**Issue by singpolyma** [mordak/Term48#5]
Either as a command or using touch screen or some other way.

**Comment by mordak** [comment]
Yes, copying is not possible at the moment (sorry). It is on the list of things to implement. In the meantime, you can write to the files in /var/clipboard/, which does the same thing as copy (and you can read from these files to get the clipboard contents).

**Comment by singpolyma** [comment]
Ooh, `/var/clipboard` is neat. Good find, thanks!
